cfe_internal/enterprise/federation/federation.cf

```diff
@@ -106,6 +108,13 @@ bundle agent config
         expression => strcmp("paused", "$(data[target_state])"),
         scope => "namespace";
 
+      "am_off"
+        expression => strcmp("off", "$(data[target_state])"),
+        scope => "namespace";
+
+    !enabled::
+      "am_off" expression => "any";
```

Comment:
Looks like this could be moved up before the `enabled::` guard as
```
"am_off" expression => or("!enabled", strcmp("off", "$(data[target_state])")
```